snakemake_executor_plugin_slurm/__init__.py

```
@@ -238,6 +331,68 @@ def __post_init__(self, test_mode: bool = False):
+                elif sacct_available and min_job_age is not None:
+                    if (
+                        min_job_age < dynamic_check_threshold
+                        and status_command == "squeue"
+                    ):
+                        self.logger.warning(
+                            f"MinJobAge is {min_job_age} seconds "
+                            f"(< {dynamic_check_threshold}s). "
+                            f"This may cause 'squeue' to report inaccurate "
+                            "job states and the status_command option may "
+                            "be unreliable. "
+                            "(Threshold is 3x status check interval: 3 × "
```

Ruff (0.14.5)

String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)